pombola/za_hansard/tests/test_akomantoso_import.py

```diff
@@ -1,56 +1,191 @@
 # -*- coding: utf-8 -*-
 
 import os
+from django.conf import settings
+import requests
 
+import socket
+from contextlib import closing
 from django.core.management import call_command
-from django.utils.unittest import skip
+from io import BytesIO
+from datetime import date
+from mock import patch
+from nose.plugins.attrib import attr
+from instances.models import Instance
 
 from instances.tests import InstanceTestCase
 from popolo_name_resolver.resolve import EntityName, recreate_entities
 
-from speeches.models import Speaker
-from pombola.za_hansard.importers.import_za_akomantoso import ImportZAAkomaNtoso, title_case_heading
+from speeches.models import Speaker, Section
+from pombola.za_hansard.importers.import_za_akomantoso import (
+    ImportZAAkomaNtoso,
+    title_case_heading,
+)
+from pombola.za_hansard.models import Source
+from pombola_sayit.models import PombolaSayItJoin
+from pombola.core.models import Person
+from popolo.models import Person as PopoloPerson
+from popolo_name_resolver.models import EntityName
 
 import logging
+
 logging.disable(logging.WARNING)
+logger = logging.getLogger(__name__)
 
 
+@attr(country="south_africa")
 class ImportZAAkomaNtosoTests(InstanceTestCase):
-
     @classmethod
     def setUpClass(cls):
-        cls._in_fixtures = os.path.join(os.path.abspath(
-            os.path.dirname(__file__)), 'test_inputs', 'hansard')
+        cls._in_fixtures = os.path.join(
+            os.path.abspath(os.path.dirname(__file__)), "test_inputs", "hansard"
+        )
         super(ImportZAAkomaNtosoTests, cls).setUpClass()
-        call_command('update_index', interactive=False, verbosity=0)
+
+        # Delete test index
+        requests.delete("http://elasticsearch:9200/pombola_test")
```

Comment:
wouldn't rebuild_index do the job here?

Reply:
For some reason `rebuild_index` just hanged on my PC when I used it instead of the delete above. I'll try it again and see if I can find out why it did that.